From 0f7acc596fab00ca88db996ccb620811d22d2cb4 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:39:57 -0400
Subject: [alsa] usb-audio: improve dmesg source grepability

Message-id: <140972823573-404-patchset-jkysela@redhat.com>
Patchwork-id: 91100
O-Subject: [RHEL 7.1 PATCH 403/442] ALSA: usb-audio: improve dmesg source grepability
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This improves messages from commit 80acefff3bc7bd53d46c16c683ab66c0ece20810.

Cc: Alexey Fisher <bug-track@fisher-privat.net>
Signed-off-by: Michal Miroslaw <mirq-linux@rere.qmqm.pl>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 82c1cf0a7fe13760baa891760015a6422d7900bb)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/usb/mixer.c b/sound/usb/mixer.c
index 0b728d8..2e4a9db 100644
--- a/sound/usb/mixer.c
+++ b/sound/usb/mixer.c
@@ -1340,12 +1340,11 @@ static void build_feature_ctl(struct mixer_build *state, void *raw_desc,
   */
  if (range > 384) {
   usb_audio_warn(state->chip,
-          "Warning! Unlikely big volume range (=%u), "
-          "cval->res is probably wrong.",
+          "Warning! Unlikely big volume range (=%u), cval->res is probably wrong.",
           range);
-  usb_audio_warn(state->chip, "[%d] FU [%s] ch = %d, "
-          "val = %d/%d/%d", cval->id,
-          kctl->id.name, cval->channels,
+  usb_audio_warn(state->chip,
+          "[%d] FU [%s] ch = %d, val = %d/%d/%d",
+          cval->id, kctl->id.name, cval->channels,
           cval->min, cval->max, cval->res);
  }
 
-- 
1.7.1